.github/workflows/ci.yml

```diff
@@ -16,10 +16,10 @@ jobs:
 
     strategy:
       matrix:
-        rust_version: [stable, "1.41.0"]
+        rust_version: [stable, "1.50.0"]
```

Comment:
Missed this before; did we use some functionality from 1.50.0 in this PR? And if we do this, we need to change the README to match.